app/controllers/users_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
   end
 
   def show
-    @user = User.find(session[:user_id])
+    @user = current_user
```

Comment:
Nice use of a helper method! I could probably use that in the sessions controller as well.